ProjectTemplates/DotNetTemplate/MonoGame.Templates.FSharp/content/MonoGame.Application.DesktopGL.FSharp/Game1.fs

```diff
@@ -0,0 +1,49 @@
+namespace MGNamespace
+
+open Microsoft.Xna.Framework
+open Microsoft.Xna.Framework.Graphics
+open Microsoft.Xna.Framework.Input
+
+type Game1 () as this =
+    inherit Game()
+ 
+    let mutable _graphics : GraphicsDeviceManager = null
```

Comment:
Can be `let graphics = new GraphicsDeviceManager(this)`.

Reply:
Also, no underscore https://fsharp.org/specs/component-design-guidelines/#-avoid-using-underscores-in-names

Reply:
> Can be `let graphics = new GraphicsDeviceManager(this)`.

Same goes for C# but for C# its not a good practice to set values of variables outside the constructor.

> Also, no underscore https://fsharp.org/specs/component-design-guidelines/#-avoid-using-underscores-in-names

Fixed.

Reply:
For reference the template I added in 2013 might be useful, let me see if I can find it...
https://github.com/MonoGame/MonoGame/pull/943/files

Reply:
I added comments at the relevant bits anyway.

Reply:
@Jjagg Yeah the let and do bindings form an implicit constructor.